### PR TITLE
Misc updates: oshadura, coffea-casa and AGC

### PR DESCRIPTION
**Pull request by oshadura**
@alexander-held please approve AGC web page updates

### CODE REVIEW

**Comment by rct225** [comment]
It's failing until Sam is successfully added.  See https://github.com/iris-hep/iris-hep.github.io/pull/1751

File: pages/projects/agc.md [hunk 1]
```diff
@@ -50,14 +52,15 @@ The Analysis Grand Challenge is being conducted during 2021‒2023, leaving enou
 - Execute second part of IRIS-HEP AGC tools soft-launch event - [AGC Tools 2022 Workshop](https://indico.cern.ch/event/1126109/)
 - Developed an analysis example based on CMS Opendata used for Analysis Grand Challenge demonstration
```
Comment (alexander-held) [inline pages/projects/agc.md:53]: Let's add

```
- Presented [first performance measurements with an AGC implementation](https://indico.cern.ch/event/1106990/contributions/4998188/) at ACAT 2022
```

as another point here.

Comment (alexander-held) [reply pages/projects/agc.md:53]: I can't seem to be able to resolve this thread, but this is done, thank you!

**Review by alexander-held — APPROVED** [review]

**Comment by oshadura** [comment]
> It's failing until Sam is successfully added. See #1751

@rct225 I removed Sam from this PR for now, please feel free to merge.

**Pull request merged**